journals/apps/journals/forms.py

```diff
@@ -15,3 +16,26 @@
     ],
     'videos/video_permissions_formset.html'
 )
+
+
+class UsersJournalAccessForm(forms.Form):
+    """
+    Form to give access for a journal to multiple users.
+    """
+
+    expiration_date = forms.DateField(
+        required=True,
+        widget=forms.widgets.DateInput(format="%d/%m/%Y"),
```

Comment:
Should not this date format should be `%m/%d/%Y`

Reply:
@zia, in real journal access admin form, the format is `"%d/%m/%Y"`.

<img width="662" alt="screen shot 2018-07-23 at 10 08 35 pm" src="https://user-images.githubusercontent.com/8234192/43091965-fbdb13a0-8ec4-11e8-8e19-b82a01428cea.png">